compatible.csv

```diff
@@ -0,0 +1,5 @@
+CKB,0.110,0.109,0.108,0.107,0.106,0.105,0.104,0.103
+Neuron
```

Comment:
```suggestion
Neuron,,,,,,,,
```

Reply:
It's blamed on GitHub if `,`s don't exist.

![image](https://github.com/nervosnetwork/neuron/assets/7271329/b1037a72-38ce-4e45-b51d-d75f51e884ef)

I think CSV has its own spec